packages/gatsby/src/utils/__tests__/__snapshots__/handle-flags.ts.snap

```diff
@@ -85,9 +85,7 @@ Object {
       "umbrellaIssue": "test",
     },
   ],
-  "message": "
-
-We're shipping new features! For final testing, we're rolling them out first to a small % of Gatsby users
```

Comment:
Result of adding checks before appending newlines to `message` in `handle-flags.ts` - it should no longer have "ghost" empty lines in front of messages